test/errors.js

```diff
@@ -13,11 +13,12 @@ t.test('errors', t => {
       t.plan(1)
 
       t.test('should print a validation error to string', t => {
```

Comment:
could you add a test that generates a validation error and then checks the status code of the response?

Reply:
Done. 

In the process I learned that the default error formatter in mercurius will use a status code of 200 when the query result is nullable, so I added a test for that as well. This behavior can be overridden with a non-default error formatter, but I think that's outside the scope of this plugin.

Reply:
nice one - cheers!